server/src/main/java/com/server/loan/calculator/service/AdminService.java

```diff
@@ -0,0 +1,17 @@
+package com.server.loan.calculator.service;
+
+import com.server.loan.calculator.model.AdminData;
+import com.server.loan.calculator.validator.AdminDataValidator;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AdminService {
+
+    private final AdminData adminData = new AdminData();
```

Comment:
would make more sense to instantiate this inside the method that you are using to return it, as later you will be getting it from database